src/main/java/net/snowflake/client/jdbc/SnowflakeBaseResultSet.java

```diff
@@ -1812,11 +1817,12 @@ public boolean isWrapperFor(Class<?> iface) throws SQLException {
     return iface.isInstance(this);
   }
 
-  private <T> Map<String, Object> prepareMapWithValues(Object object, Class<T> type)
-      throws SFException {
-    if (object instanceof JsonSqlInput) {
+  private <T> Map<String, Object> prepareMapWithValues(
```

Comment:
where are we using type?

Reply:
It was already there, in line 1830 to check if it's assignable to SQLData